go.mod

```diff
@@ -1,11 +1,22 @@
 module github.com/iotexproject/iotex-proto
 
 require (
-	github.com/golang/protobuf v1.3.1
-	github.com/grpc-ecosystem/grpc-gateway v1.9.0
-	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
-	golang.org/x/sys v0.0.0-20190502175342-a43fa875dd82 // indirect
-	golang.org/x/text v0.3.2 // indirect
-	google.golang.org/genproto v0.0.0-20190502173448-54afdca5d873 // indirect
-	google.golang.org/grpc v1.20.1
+	cloud.google.com/go v0.43.0 // indirect
+	github.com/golang/protobuf v1.3.2
```

Comment:
@koseoyoung can you check if the so many dependency change is because of protobuf upgrade?